src/operator/scan.ts

```diff
@@ -13,14 +13,15 @@ import {errorObject} from '../util/errorObject';
  * @returns {Obervable} An observable of the accumulated values.
  */
 export function scan<T, R>(accumulator: (acc: R, x: T) => R, seed?: T | R): Observable<R> {
-  return this.lift(new ScanOperator(accumulator, seed));
+  let _this: Observable<T> = this;
```

Comment:
Again, for this sort of thing, I fail to see the benefit to our end users. It's just going to add to the transpiled output, I believe.